sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -775,6 +872,88 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3.1 Manual hyperparameter sweeping for models from Hugging Face (Preview)\n",
+    "\n",
+    "Similar to how you can use any model algorithm from Hugging face transformers library for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of model algorithms supported supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and model algorithms from [Hugging Face](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers).\n",
+    "\n",
+    "In this example, we sweep over `microsoft/beit-base-patch16-224-pt22k-ft22k`, `facebook/deit-base-patch16-224`, `seresnext`, and `resnet50` models choosing from a range of values for learning_rate, number_of_epochs, etc., to generate a model with the optimal 'accuracy'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "image_classification_job = automl.image_classification(\n",
+    "    compute=compute_name,\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=\"label\",\n",
+    "    primary_metric=ClassificationPrimaryMetrics.ACCURACY,\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "image_classification_job.set_limits(\n",
+    "    timeout_minutes=60,\n",
```

Comment:
this is not enough to finish the sweep - maybe we can remove it or increase it to a couple of hours if removing it poses issues for e2e tests